db/forecast.spec.js

```diff
@@ -0,0 +1,60 @@
+import Forecast, { db } from "./forecast";
+import { current } from "../db/fixtures/forecast";
+
+describe("Forecast Model", () => {
+	beforeAll(async () => {
+		await db.sync({ force: true });
+	});
+
+	beforeEach(async () => {
+		await Forecast.destroy({ where: {}, truncate: true });
+	});
+
+	describe("#getByZipAndTSandMaxAge", () => {
+		it("should return a fresh record from the database if its exists and within age range", async () => {
+			// Arrange
+			await Forecast.bulkCreate([current]);
```

Comment:
`note`: Must use `bulkCreate` instead of `bulk` so that the `updatedAt` time can be set.